extensions/amp-ima-video/0.1/validator-amp-ima-video.protoascii

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: { #amp-ima-video
```

Comment:
Hi @shawnbuso , I work on AMP validation and going to look at just the validation portion of your PR. For more on validation you can see [Contributing Validator Rules](https://github.com/ampproject/amphtml/blob/master/contributing/component-validator-rules.md).

We recently simplified how to specify the script tag for custom-elements in the protoascii. Could you please update your script section here to the following:

```
tags: {  # amp-ima-video
  html_format: AMP
  tag_name: "SCRIPT"
  satisfies: "amp-ima-video extension .js script"
  requires: "amp-ima-video"
  extension_spec: {
    name: "amp-ima-video"
    allowed_versions: "0.1"
    allowed_versions: "latest"
  }
  attr_lists: "common-extension-attrs"
}
```